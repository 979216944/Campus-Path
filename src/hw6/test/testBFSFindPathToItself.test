# Test BFS finding a path,
# from a character in the dataset(flowers.tsv) to itself.

# Load graph
LoadGraph flowers flowers.tsv

# Find a path to itself
FindPath flowers Aconitum Aconitum
FindPath flowers Alchemilla Alchemilla
FindPath flowers Quince Quince
FindPath flowers Gaillardia Gaillardia