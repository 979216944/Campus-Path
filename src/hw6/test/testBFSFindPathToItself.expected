# Test BFS finding a path,
# from a character in the dataset(flowers.tsv) to itself.

# Load graph
loaded graph flowers

# Find a path to itself
path from Aconitum to Aconitum:
path from Alchemilla to Alchemilla:
path from Quince to Quince:
path from Gaillardia to Gaillardia: